### PR TITLE
Server: Extract raw api to be implemented in Project1 into raw_api.go

### PR DESCRIPTION
**Pull request by pupillord**
### What problem does this PR solve?
Rearrange code of project1

### What is changed
Extract raw api to be implemented in Project1 into raw_api.go

Issue :#283

### CODE REVIEW

**Review by ngaut — APPROVED** [review]
LGTM

File: kv/server/server.go [hunk 1]
```diff
@@ -35,26 +35,8 @@ func NewServer(storage storage.Storage) *Server {
...
-	// Your Code Here (1).
-	return nil, nil
-}
-
-func (server *Server) RawScan(_ context.Context, req *kvrpcpb.RawScanRequest) (*kvrpcpb.RawScanResponse, error) {
-	// Your Code Here (1).
-	return nil, nil
-}
+// Project1:
+// Implement raw api in raw_api.go
```
Comment (tisonkun) [inline kv/server/server.go]: We don't need this line then.

We should update the document and mention "related file" as described in #275 .

File: kv/server/raw_api.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+func (server *Server) RawDelete(_ context.Context, req *kvrpcpb.RawDeleteRequest) (*kvrpcpb.RawDeleteResponse, error) {
+	// Your Code Here (1).
+	return nil, nil
+}
+
+func (server *Server) RawScan(_ context.Context, req *kvrpcpb.RawScanRequest) (*kvrpcpb.RawScanResponse, error) {
+	// Your Code Here (1).
+	return nil, nil
+}
```
Comment (tisonkun) [inline kv/server/raw_api.go]: Add a new line or just run `go fmt`.

**Review by tisonkun — CHANGES_REQUESTED** [review]
Thanks for your contribution @pupillord ! Comments inline.

Also it would be better if you can elaborate more on functions in `raw_api.go`

**Review by tisonkun — APPROVED** [review]
LGTM. Thanks for your contribution!

Merging...

**Pull request merged**